BLOG-POST-TEMPLATE.md

```diff
@@ -184,15 +186,25 @@ If you want an easier way to make tables in markdown, you can also try this
 
 ## Metadata
 
+We can use [OpenGraph meta tags](https://ogp.me/) for social sharing. Hugo has
+an embedded OpenGraph template:
+https://gohugo.io/templates/embedded/#open-graph.
+[Hugo OpenGraph embedded template source code](https://github.com/gohugoio/hugo/blob/master/tpl/tplimpl/embedded/templates/opengraph.html).
+
 At this time, the blog generates metadata in the following ways:
 
-- the `title` parameter in the blog post head generates the `<title></title>`
+- The `title` parameter in the blog post head generates the `<title></title>`
   tag
-- [OpenGraph meta tags](https://ogp.me/) are generated for social sharing
+- The meta description in the blog post head is generated by the `description`
+  in the [blog post head](#blog-head). If no `description` is set in the front
+  matter of the post, there will not be a meta description tag.
+- [OpenGraph meta tags](https://ogp.me/):
```

Comment:
Should we add any of the following?

* `og:site_name` - it looks like it is set to the site title.
* `og:locale` - set to `locale` (although slightly modified)
* `og:audio` - set to `audio`, should be a list of paths
* `og:video` - set to `video`, should be a list of paths
* `og:see_also` - uses taxonomy. I am not sure exactly when it would show without testing
* `article:tag` - I believe this will be set to the tags from above.

There is also `fb:*`, but I doubt we would use that.

Reply:
I guess we could, but I didn't see a strong need to list out *all* of them since they're listed at https://ogp.me/. It seemed redundant to just copy and paste them into this doc. It also seems like the only ones marketing really cares about is `title` and `description`. But I guess @christophermluna can confirm whether he thinks we'll ever use some of the ones you listed.

Reply:
My thought was that we would describe how they are set, which seems like valuable information and not something that https://ogp.me/ would have. For instance, if we add a video to a blog post in the future,  `og:video` would be useful. The tags and see-also may be useful for SEO, etc.

Reply:
I'll add the first 4, but I'll create a separate story for:
1. `og:see_also`: We don't currently use `series` for blog posts, nor do I know how to implement that right now.
2. `article:tag`: As of right now, this does not populate the tags. For example, I'd expect the 4 tags at the bottom of https://blog.maxmind.com/2024/07/impact-of-consumer-privacy-networks-on-geolocation-online-security-and-business-tactics/ to be populated in the `<meta property"article:tag">`, but I don't see this metadata populated at all.

Reply:
https://www.pivotaltracker.com/story/show/188422897